src/sdk/model/__tests__/company.test.int.ts

```diff
@@ -113,3 +113,32 @@ test('Can refresh company', async() => {
     });
   });
 });
+
+test('Can set logo', async() => {
```

Comment:
if we're going to have integration tests for this, I'd prefer that we put back the company's original logo in the end. We have to remember that customers will be using this in the future and it would be annoying if it removes/updates their logo if they're running these tests.